ui/package.json

```diff
@@ -89,7 +89,6 @@
     "eslint-plugin-vue": "^7.0.0",
     "less": "^3.0.4",
     "less-loader": "^5.0.0",
-    "node-sass": "^4.0.0",
     "sass": "^1.49.9",
```

Comment:
We're using the dart-sass already (see https://github.com/sass/node-sass#node-sass)